qa-tests-backend/src/test/groovy/IntegrationsSplunkViolationsTest.groovy

```diff
@@ -83,7 +82,6 @@ class IntegrationsSplunkViolationsTest extends BaseSpecification {
     }
 
     @Category(Integration)
-    @Ignore("ROX-11138")
```

Comment:
Previously it was[ `@IgnoreIf({ Env.CI_JOBNAME.contains("postgres") })`](https://github.com/stackrox/stackrox/pull/1878/files) but I'm fine with enabling this test on postgres if it passes

Reply:
glancing at it, I think it should pass with postgres on.  I recommend adding the postgres ci label to find out